layouts/section/job-posting.fr.html

```diff
@@ -1,144 +1,151 @@
 {{ define "main" }}
-<div class="container">
-  <section class="content">
-    <section class="section page--outer-container-padding">
+<div class="job-postings-container">
+  <section>
+    <section class="padding">
       <div class="row">
         <div class="col-sm-10 col-sm-offset-1 col-xs-12">
-          <h2 class="section--title">{{ i18n "hiring" }}</h2>
-          {{ .Content }}
+          {{ if .Params.leverId }}
```

Comment:
If there is no lever ID we're going to have a post that skips the `h1` and goes straight to `h2`.

Reply:
This if seems to be in a different position in the english version?

Reply:
Sorry! forgot to change the fr version of it, en version is correct. Now for both EN and FR, the "apply now" button will only exist if there is a leverId